sys-libs/libxcrypt/libxcrypt-4.4.28.ebuild

```diff
@@ -114,19 +123,49 @@ src_configure() {
 	multibuild_foreach_variant multilib-minimal_src_configure
 }
 
+get_xcprefix() {
+	if is_cross; then
+		echo "${EPREFIX}/usr/${CTARGET}"
+	else
+		echo "${EPREFIX}"
+	fi
+}
+
 get_xclibdir() {
-	printf -- "%s/%s/%s\n" \
+	printf -- "%s/%s/%s/%s\n" \
+		"$(get_xcprefix)" \
 		"$(usex split-usr '' '/usr')" \
 		"$(get_libdir)" \
 		"$(usex system '' 'xcrypt')"
 }
 
+get_xcincludedir() {
+	printf -- "%s/%s/include/%s\n" \
+		"$(get_xcprefix)" \
+		"$(usex split-usr '' '/usr')" \
+		"$(usex system '' 'xcrypt')"
+}
+
+get_xcmandir() {
+	printf -- "%s/%s/share/man\n" \
+		"$(get_xcprefix)" \
+		"$(usex split-usr '' '/usr')"
+}
+
+get_xcpkgconfigdir() {
+	printf -- "%s/%s/%s/pkgconfig\n" \
+		"$(get_xcprefix)" \
+		"$(usex split-usr '' '/usr')" \
+		"$(get_libdir)"
+}
+
 multilib_src_configure() {
 	local -a myconf=(
 		--disable-werror
 		--libdir="${EPREFIX}"$(get_xclibdir)
```

Comment:
Double prefix again -- just `$(get_xclibdir)` (`get_xclibdir` is guaranteed to return something prefixed with `${EPREFIX}`, so prepending it with `${EPREFIX}` is doubling it)